src/charm.py

```diff
@@ -240,23 +275,23 @@ def _create_discourse_environment_settings(self):
 
         saml_config = self._get_saml_config()
         for key in saml_config:
-            pod_config[key] = saml_config[key]
+            pod_config.update({key: saml_config[key]})
```

Comment:
what about `pod_config.update(self._get_saml_config())`, that would negate the need for the for loop

Reply:
Done